frontend/src/pages/InducteeSignUpPage/components/SignUpForm/index.tsx

```diff
@@ -0,0 +1,274 @@
+                </Grid>
+              </Grid>
+            </Grid>
+
+            <Grid item>
+              <Field
+                component={TextField}
+                fullWidth
+                name='email'
+                label='Email Address'
+              />
+            </Grid>
+
+            <Grid item>
+              <Field
+                component={TextField}
+                fullWidth
+                name='password'
+                type='password'
+                label='Password'
+              />
+            </Grid>
+
+            <Grid item>
+              <Field
+                component={TextField}
+                fullWidth
+                name='confirmPW'
+                type='password'
+                label='Confirm Password'
+              />
+            </Grid>
+
+            <Grid item>
+              <Grid container direction='row' spacing={2}>
+                <Grid item xs={8}>
+                  <MajorDropdownField
+                    name='major'
+                    label='Major'
+                    fullWidth
+                  />
+                </Grid>
+
+                <Grid item xs={4}>
+                  <YearDropdownField
+                    name='graduationYear'
+                    label='Grad Year'
+                    fullWidth
+                  />
+                </Grid>
+              </Grid>
+            </Grid>
+
+            <Grid item>
+              <h3>Review your coursework</h3>
+              <Typography>Description.</Typography>
+            </Grid>
+            <Grid item>
+              <img
+                className={classes.logo}
+                src={FOUR_YEAR_PLAN}
+                alt='Coursework Requirements'
+              />
+            </Grid>
+            <Grid item>
+              <Grid container direction='row' alignItems='center'>
+                <Grid item xs={1}>
+                  <Field
+                    type='checkbox'
+                    component={Checkbox}
+                    name='courseRequirement'
+                    color='primary'
+                  />
+                </Grid>
+                <Grid item>
+                  <Typography>I understand</Typography>
+                </Grid>
+              </Grid>
+            </Grid>
+
+            <Grid item>
+              <h3>Select an info session</h3>
+            </Grid>
+
+            <Grid item>
+              <InfoSessionDropdownField
+                name='infoSession'
+                label='Info Session'
+                fullWidth
+              />
+            </Grid>
+
+            <Grid item>
+              <h3>Buddy related (TBD)</h3>
+            </Grid>
+
+            <Grid item>
+              <h3>Newsletter Subscription</h3>
+            </Grid>
+            <Grid item>
+              <Grid container direction='row' alignItems='center'>
+                <Grid item xs={1}>
+                  <Field
+                    type='checkbox'
+                    component={Checkbox}
```

Comment:
We can probably use [this](https://stackworx.github.io/formik-material-ui/docs/api/material-ui#checkboxwithlabel) for checkboxes, just so it's more convenient to add the label, as opposed to having to add a whole separate component below for the label ourselves